08LessonEight/in-class-project/homework.html

```diff
@@ -0,0 +1,76 @@
+        </p>
+    </div>
+  </section>
+  <section>
+    <div class=" clearfix hands">
+      <img src=".//img/keyboard.jpg" width="325px" height="175px">
+        <h2 style="font-size:18px">Morbi eu metus ut libero laoreet vulpurtate</h2>
+          <p style="font-size:11px" width="90px" height="200px">Last week, as the musical guest on “Saturday Night Live,” she performed her songs “Nobody Really Cares If You Don't Go to the Party” and “Pedestrian at Best.” Recently, the National and Bonnie “Prince” Billy performed Grateful Dead songs on “Colbert” and “CBS This Morning.”
+          </p>
+            <a href="https://www.utexas.edu">Read More</a>
+    </div>
+  </section>
+  <section>
+    <div class="clearfix small1">
+      <img src=".//img/laptop.jpg" width="130px" heigh="80px">
+      <h2 style="font-size:18px">Suspendisse a sagittis</h2>
+        <p style="font-size:11px">Contours, a new feature where we take you on tour with some of our favorite artists, kicked off earlier this month with Australian singer/songwriter Courtney Barnett. </p>
+    </div>
+  </section>
+  <section>
+    <div class="clearfix small2">
+      <img src=".//img/desk.jpg" width="130px" heigh="80px">
+      <h2 style="font-size:18px">Morbi eu metus ut libero</h2>
+        <p style="font-size:11px"> Photographer Charlotte Zoller followed along on Barnett's recent U.S. tour as she ventured through wine-soaked Napa, took the Southern route through Los Angeles and San Diego, got weird in Austin, and finally ended up in New Orleans.
+        </p>
+    </div>
+  </section>
+  <section>
+    <div class="clearfix small3">
+      <img src=".//img/window.jpg" width="130px" heigh="80px">
+      <h2 style="font-size:18px">Pellentesque magna dolor</h2>
+        <p style="font-size:11px" > Photographer Charlotte Zoller documents Australian singer/songwriter Courtney Barnett's recent tour through the American Southwest.
+        </p>
+    </div>
+  </section>
+    <hr class="clear">
+  <section class="two">
+    <div class="clear list1" style="page-break-after:auto">
+      <h3>Why Profession?</h3>
+        <ul>
+          <li>Clean and Corporate webdesign</li>
+          <li>Well coded HTML template</li>
+          <li>Multiple Layout Options</li>
+        </ul>
+    </div>
+
+    <div class="list2" style="border-left:1px solid #000;height:100px">
+      <h3>Main Features</h3>
+        <ul>
+          <li>Bootstrap 3x</li>
+          <li>Super lightning page speed</li>
+          <li>well documented code</li>
+        </ul>
+    </div>
+    <div class="list3" style="border-left:1px solid #000;height:100px">
+      <h3>Transparent Pricing</h3>
+        <p>Are you interested in our service? Dont hesitate to contact us.
+        </p>
+        <a href="https://www.utexas.edu">Check Pricing</a>
+        <!--Put Link Here Check Pricing-->
+    </div>
+  </section>
+  <hr class="clear">
+</body>
+</html>
```

Comment:
if you're going to write styling in your html, i'd rather you do ALL of your styling in html, rather than split it between html and css. if something is broken, it's a lot harder to find the mistake if you need to check multiple files